docs/hybrid-slurm-cluster/requirements.txt

```diff
@@ -0,0 +1,2 @@
+addict~=2.0
+google-cloud-pubsub~=2.0
```

Comment:
In a fresh environment that has these dependencies installed I still get an error for `httplib2`, `google-api-python-client`, and `yaml`.

Add the following to this file (or something of the sort):

```
google-api-python-client==2.61.0
httplib2==0.20.4
PyYAML==6.0
```

Reply:
Done